examples/rotator_test_light.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
```

Comment:
So this is a script not a notebook... shouldn't it be in ts_externalscripts, or maybe in a different place?